src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/CodeStyle/CSharpIdeCodeStyleOptions.cs

```diff
@@ -80,32 +80,33 @@ public override bool Equals(object? obj)
 
     public bool Equals([AllowNull] CSharpIdeCodeStyleOptions other)
         => other is not null &&
-           Common.Equals(Common) &&
```

Comment:
@tmat i presumet he old logic here wasn't intentional (e.g. testing properties against themselves?)

Reply:
Definitely not. Thanks for fixing.